Cmdline/Action/Install.cs

```diff
@@ -129,7 +129,7 @@ public int RunCommand(CKAN.KSP ksp, object raw_options)
 
                 string message = String.Format("Too many mods provide {0}. Please pick from the following:\n", ex.requested);
 
-                int result = -1;
+                int result;
```

Comment:
Oh, of course! I was thinking of another reality where exceptions inside a `try` that weren't explicitly caught were instead suppressed. :)